.env.example

```
@@ -9,6 +9,8 @@ ANTHROPIC_FREE_MODEL=claude-haiku-4-5-20251001
 ANTHROPIC_PREMIUM_MODEL=claude-sonnet-4-5-20250929
 ANTHROPIC_PREMIUM_MODELS=claude-sonnet-4-5-20250929,claude-opus-4-5-20251101
 ANTHROPIC_BYOK_MODEL=claude-haiku-4-5-20251001
+ANTHROPIC_MODEL=                         # Legacy fallback model ID (optional)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)